membership_withdrawl/__openerp__.py

```diff
@@ -0,0 +1,40 @@
+# -*- encoding: utf-8 -*-
+# Python source code encoding : https://www.python.org/dev/peps/pep-0263/
+##############################################################################
+#
+#    OpenERP, Odoo Source Management Solution
+#    Copyright (c) 2015 Antiun Ingeniería S.L. (http://www.antiun.com)
+#                       Antonio Espinosa <antonioea@antiun.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': "Membership withdrawal",
+    'category': 'Association',
+    'version': '1.0',
+    'depends': [
+        'membership',
```

Comment:
There is a dependency in res_partner_view.xml view:
```
<field name="inherit_id" ref="membership.view_partner_form"/>
```
Maybe this XML file is not named correctly? This view inherit from a view that inherits from res_partner_view, declared in membership. If module 'membership is not installed, this view will not find ```associate_member``` field:
```
<field name="associate_member" position="after">
```